notifier/notifier.go

```diff
@@ -383,25 +391,25 @@ func (n *Notifier) Stop() {
 
 // Describe implements prometheus.Collector.
 func (n *Notifier) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
This no longer needs to be a prometheus.Collector, register the metrics in an `init()`.